pyincore/analyses/epfrestoration/epfrestoration.py

```diff
@@ -169,7 +211,27 @@ def get_spec(self):
                     'required': True,
                     'description': 'DFR3 Mapping Set Object',
                     'type': ['incore:dfr3MappingSet'],
+                },
+                {
+
+                    'id': 'restoration_table',
```

Comment:
Do we always assume we will have a discrete restoration table exist? I.e. is this analysis always going to calculate both continuous and discrete?

Reply:
That's a good question. We can discuss it in today's dev meeting or merge this and update this later.